src/resolution/resolution.service.js

```diff
@@ -12,22 +12,25 @@ export default class ResolutionService {
 
   addResolution = async (body, doctorUserId) => {
     const doctor = await this.doctorService.getDoctorByUserId(doctorUserId);
-
-    const data = body;
-    data.doctor_name = doctor.name;
-    data.doctor_specialization = doctor.specialization;
-
-    data.patient_id = body.patientId;
-
-    if (!body.ttl) {
-      data.ttl = defaultTTL;
-    } else {
-      data.ttl = body.ttl;
-    }
-
-    const resolution = await this.repository.add(data);
-
-    return resolution;
+    const { resolution, patientId, ttl } = body;
+    const {
+      name: doctorName,
+      id: doctorId,
+      specialization: doctorSpecialization,
+    } = doctor;
+
+    const data = {
+      resolution,
+      patientId,
+      doctorName,
+      doctorId,
+      doctorSpecialization,
+      ttl: ttl || defaultTTL,
```

Comment:
If we don't get any **TTL** value from the client side, we'll use a **default TTL** value from config (it is currently **`-1`**, but it can be changed)

Reply:
:white_check_mark: